src/test/java/edu/ucsb/cs156/example/controllers/UCSBSubjectsControllerTests.java

```diff
@@ -24,7 +24,7 @@
 import java.util.Map;
 import java.util.Optional;
 
-import com.mongodb.DuplicateKeyException;
+// import com.mongodb.DuplicateKeyException;
```

Comment:
Professor says to leave the comments there for next quarters class in case they need to deal with mongo and it would be easier for the comments to still be there.